juriscraper/opinions/united_states/state/minn.py

```diff
@@ -59,8 +59,19 @@ def _process_html(self) -> None:
         """
         self.html = self._download({"params": self.params})
 
+        # This warning is useful for backscraping
+        results_number = self.html.xpath(
```

Comment:
Fail here in this range: 2024-05-15 to 2024-05-21

There are no results, you are taking the first element [0] without first checking that the element exists

Reply:
I have updated the PR to account for this, please check again @quevon24